tests/external/scripts/lapack/ex1.py

```diff
@@ -0,0 +1,156 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+# pylint: disable=unused-variable
+# TODO: Remove disable unused variable when #927 is fixed
+# > Usage:
+#
+#   pyccel test.py -t
+#   gfortran test.f90 -lblas -llapack
+#   ./a.out
+
+# TODO: - assert
+
+from scipy.linalg.lapack import dgbtrf
+from scipy.linalg.lapack import dgbtrs
+
+from scipy.linalg.lapack import dgetrf
+from scipy.linalg.lapack import dgetrs
+#from scipy.linalg.lapack import dgecon
+
+#from scipy.linalg.lapack import dgetri
+
+from numpy import zeros, empty
+
+def test_1():
+    n   = 25
+    ml  = 1
+    mu  = 1
+    lda = 2 * ml + mu + 1
+
+    a = zeros((lda,n))
+    b = zeros((1,n))
+
+    b[0]   = 1.0
+    b[n-1] = 1.0
+
+    # Superdiagonal, Diagonal, Subdiagonal
+    m = ml + mu
+    a[m-1,1:n] = -1.0
+    a[  m,0:n] =  2.0
+    a[m+1,0:n-1] = -1.0
+
+    info = -1
+    ipiv = zeros(n, 'int')
```

Comment:
Do we still need these dummy declarations in order to use the macros below?

Reply:
Yes, this PR just takes care of ensuring that the input and output match what is described by the macro description. The macro is unravelled to a function call with no return arguments so these objects still need to exist beforehand. This is described in issue #927 I believe @rmahjoubi is taking care of that

Reply:
Excellent, thanks